utils/Cargo.toml

```diff
@@ -18,3 +21,5 @@ miden-objects = { workspace = true }
 serde = { version = "1.0", features = ["derive"] }
 tracing = { workspace = true }
 tracing-subscriber = { workspace = true }
+tracing-forest = { version = "0.1", optional = true, features = ["chrono"] }
+miden-node-test-macro = { path = "../test-macro", optional = true }
```

Comment:
Do we still need `miden-node-test-macro` dependency here?

Reply:
removed